redpanda/templates/configmap.yaml

```diff
@@ -30,6 +30,21 @@ metadata:
   {{- toYaml . | nindent 4 }}
 {{- end }}
 data:
+{{- if (include "redpanda.semver" . | semverCompare ">=22.1.4") }}
```

Comment:
I believe the correct version would be `22.1.1`: https://github.com/redpanda-data/redpanda/releases/tag/v22.1.1

Reply:
Fixed in [519e4f1](https://github.com/redpanda-data/helm-charts/pull/117/commits/519e4f193b60c4e2410f2b1042c3eb2d958f5ea7)